Round_1_query_4.sql

```diff
@@ -0,0 +1,6 @@
+SELECT
+host_id,
+COUNT(neighbourhood) as count_neighbourhood
```

Comment:
This seems incomplete at the moment -> you're returning all hosts with more than one listing neighbourhood, but the requirement is to only return a total number of hosts passing the criteria (having listings at more than one location)